### PR TITLE
[2/N][multi quorum ejection] Fetch operator quorum opt-in/out info

### PR DESCRIPTION
**Pull request by jianoaix**
## Why are these changes needed?
This is part of the work to compute operator nonsigning rate for ejection in multi quorum.

Roughly the 50% of the second step in https://docs.google.com/document/d/13nxDXTAxRhT18qZrEkTtS2F7CxjlM0qA-2EGim1bu1k/edit

## Checks

- [x] I've made sure the lint is passing in this PR.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [x] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: disperser/dataapi/subgraph_client.go [hunk 1]
```diff
@@ -51,6 +51,19 @@ type (
...
 	}
+	OperatorQuorum struct {
+		Operator      string
+		QuorumNumbers []byte
+		BlockNumber   uint64
+	}
+	OperatorQuorumEvents struct {
+		// Mapping from operator address to a list of sorted events (ascending by BlockNumber)
+		// where the operator was added to quorums.
+		AddedToQuorum map[string][]*OperatorQuorum
```
Comment (ian-shim) [inline disperser/dataapi/subgraph_client.go:62]: nit: start comment with the name of the variable. `AddedToQuorum is a mapping from...`

same below

File: disperser/dataapi/subgraph_client_test.go [hunk 2]
```diff
@@ -411,3 +446,59 @@ func assertGasFees(t *testing.T, gasFees *dataapi.GasFees) {
...
+	addedMap := result.AddedToQuorum
+	assert.Equal(t, 2, len(addedMap))
+	// Quorum events for operator-1.
+	added1, ok := addedMap["operator-1"]
+	assert.True(t, ok)
+	assert.Equal(t, 2, len(added1))
+	assert.Equal(t, "operator-1", added1[0].Operator)
+	assert.Equal(t, uint64(80), added1[0].BlockNumber)
+	assert.Equal(t, 2, len(added1[0].QuorumNumbers))
+	assert.Equal(t, uint8(48), added1[0].QuorumNumbers[0])
```
Comment (ian-shim) [inline disperser/dataapi/subgraph_client_test.go:467]: can you explain why this is 48 instead of 01?

**Review by ian-shim — APPROVED** [review]
one question to understand the output from gql, but lgtm

File: disperser/dataapi/subgraph_client.go (hunk 1 above)
Comment (jianoaix) [reply disperser/dataapi/subgraph_client.go:62]: Done

File: disperser/dataapi/subgraph_client_test.go (hunk 2 above)
Comment (jianoaix) [reply disperser/dataapi/subgraph_client_test.go:555]: Added

**Pull request merged**